### DISCUSSION TITLE
EvalAI scoring interpretation

### DISCUSSION THREAD

**Discussion by Leaderleade** [empriselab/RCareWorld#132] [Help - Setting up / Configuring] [answered]
Hello there,

Recently I have submitted a sample docker file, the unmodified docker template to EvalAI and got the result below:
{
  "Metric1": 32,
  "Metric2": 51,
  "Metric3": 17,
  "Total": 63
}

I do not understand the meanings of the metrics, and I am unsure if this result distinguishes between Track 1 and Track 2. It seems that the template Docker just ran the two test Python files. I am unable to relate the EvalAI result to the marking scheme of either track because in the ~/template/docker-template/output folder, I received the following:
{
    "PickUpGarmentScore": 0
    "PutOneWristInOneSleeveScore": 0
    "PullGownToLeftShoulderScore": 0
    "PullGownToRightShoulderScore": 0
    "OverallDressingScore": 0
    "TotalScore": 0
} 

Could you please provide clarification?

Thanks

**Comment by shxingch** [comment, marked as answer]
Hi @Leaderleade! We have rerun your submission file and your submission is now visible on the public leaderboard!
The grades for Track 1 and Track 2 can be found in the following JSON files:

- Track 1: $CONTAINER_ID:/root/.config/unity3d/RCareWorld/BathingPlayer/spongeScore.json
- Track 2: $CONTAINER_ID:/root/.config/unity3d/RCareWorld/DressingPlayer/dressingScore.json

Feel free to check these files to see your grades.
